.github/workflows/run_g_tests.yml

```diff
@@ -42,7 +42,7 @@ jobs:
 
     - name: Run G Tests
       run:
-        python ${{runner.workspace}}/measurementlink-labview/Source/Tests/run_tests.py --labview-path "C:\Program Files\National Instruments\LabVIEW ${{ matrix.version }}\LabVIEW.exe" --port-number ${{ matrix.port }}
```

Comment:
Couldn't effectively use `RelativePath` from [actions/Checkout](https://github.com/actions/checkout/blob/main/README.md), because the `$GITHUB_WORKSPACE` is `C:\ar\_work\measurement-plugin-labview\measurement-plugin-labview`.

```
    # Relative path under $GITHUB_WORKSPACE to place the repository
    path: ''
```